### PR TITLE
Added details for @shagunsodhani's talk

### PR DESCRIPTION
**Pull request by shagunsodhani**

### CODE REVIEW

**Comment by Dawny33** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -18,7 +18,8 @@ All talks presented at PyData Delhi 2017:
 |11|Difficulties in working with the government data and how can tech help us resolve it| Ankita Mathur| - | - |_in process_|
 |12|Using RNNs to implement a Full Adder using PyTorch| Madhur Tandon| - | - |_in process_|
 |13|Deriving insights in minutes from serverless IoT Data Pipeline| Narendran R| - | - |_in process_|
-|14|Machine Learning Architectures | Shagun Sodhani| - | @shagunsodhani  |_in process_|
+|14|Machine Learning Architectures | Shagun Sodhani| [Machine Learning Architectures](http://slides.com/shagunsodhani/pydata2017#/) | [@shagunsodhani](https://twitter.com/shagunsodhani)
+  |_in process_|
```
Comment (manojpandey) [inline README.md:22]: not next line :/ 
L22 should be included with L21

Difference:

|--|--|--|--|--|--|
|--|--|--|--|--|--|
|14(old)|Machine Learning Architectures | Shagun Sodhani| - | @shagunsodhani  |_in process_|
|14|Machine Learning Architectures | Shagun Sodhani| [Machine Learning Architectures](http://slides.com/shagunsodhani/pydata2017#/) | [@shagunsodhani](https://twitter.com/shagunsodhani)
  |_in process_|

**Review by manojpandey — COMMENTED** [review]
please see

File: README.md (hunk 1 above)
Comment (manojpandey) [reply README.md:22]: @Dawny33 😠

Comment (shagunsodhani) [reply README.md:22]: @manojpandey https://github.com/pydatadelhi/delhi2017/pull/6

**Pull request merged**